frontend/src/libs/components/input/input.tsx

```diff
@@ -42,19 +48,36 @@ const Input = <T extends FieldValues>({
     hasError && styles.error,
   ];
 
+  const toggleShowPassword = useCallback(
+    (event: React.MouseEvent<HTMLElement>): void => {
+      event.preventDefault();
```

Comment:
can we remove prevent default? for which purpose it's used here

Reply:
When I remove it click event bubbles up and triggers form as touched, so all field becomes red